### DISCUSSION TITLE
想简单的编译一下源码请问有知道用什么软件能打开alist这个文件吗

### DISCUSSION THREAD

**Discussion by xiaowang7212** [AlistGo/alist#1107] [Q&A]
想简单的编译一下源码请问有知道用什么软件能打开alist这个文件吗

**Comment by anwen-anyi** [comment]
自行编译文档：**https://alist-doc.nn.ci/docs/install/source-code**
就这么两句话..没啥难度..`golang>=1.17` 这里记得升级1.18~ 1.17不行了

**Reply by xiaowang7212** [reply]
好滴 谢谢  我试试·

**Reply by xiaowang7212** [reply]
唉算了，我用宝塔·不会玩·
